contracts/src/LightClient.sol

```diff
@@ -207,17 +228,16 @@ contract LightClient is Initializable, OwnableUpgradeable, UUPSUpgradeable {
 
         blocksPerEpoch = numBlockPerEpoch;
 
+        maxStateHistoryAllowed = 10 * 7166; //set the max block states recorded to be 10 days of
```

Comment:
I wonder how the 7166 was computed.

Reply:
That's the number of ethereum blocks per day - which is generally roughly one block per second but i got that figure from here https://ycharts.com/indicators/ethereum_blocks_per_day#:~:text=Ethereum%20Blocks%20Per%20Day%20(I%3AEBC)&text=Ethereum%20Blocks%20Per%20Day%20is,0.21%25%20from%20one%20year%20ago.

Reply:
The block time is 12s which would imply 5 * 60 * 24 = 7200 blocks but it seems there can be empty slots. I think we should use 7200 instead.

But I realize now that this needs to adjusted for the blockchain if the light client contract is deployed on an L2 (something we need to do soon). And if a blockchain has irregular block times we would have to do it differently. Sorry I need to think about this for a bit.

Reply:
Yea there was a typo in my original response - i meant to write one block per 12 seconds but yea i can use that figure instead.

Reply:
https://github.com/EspressoSystems/espresso-sequencer/pull/1670/commits/9cf843e997cc504a71f4d6cad4bf74688f542389